cmd/edge-orchestration/main.go

```diff
@@ -70,6 +71,9 @@ const (
 	cipherKeyFilePath = edgeDir + "/user/orchestration_userID.txt"
 	deviceIDFilePath  = edgeDir + "/device/orchestration_deviceID.txt"
 	mnedcServerConfig = edgeDir + "/mnedc/client-config.yaml"
+
+	mqttBrokerURL = "ec2-34-228-146-136.compute-1.amazonaws.com" //**Hardcoded URL and port, will be replaced by the servivce app in incremental commit
+	mqttport      = 1883
```

Comment:
Future we should migrate to MQTT over SSL for added security [8883 port]

Reply:
Yes, Once the basic functonality is working , SSL using certificates will be added. Thanks